.github/workflows/build-wasm.yml

```
@@ -0,0 +1,105 @@
+on:
+  push:
+    tags:
+      - '*'
+  pull_request:
+    paths:
+      - ".github/workflows/build-wasm.yml"
+name: Build&Publish WASM
+concurrency:
+  group: '${{ github.workflow }}-${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+permissions:
+  id-token: write
+  contents: read
+jobs:
+  build-runtime-wasms:
+    environment: production # It needs the production GCP credentials to upload WASM
+    # Benchmark (with cache hit)
+    # ubuntu-latest 50 min
+    # ubuntu-latest-4-cores 34 min
+    runs-on:  ubuntu-latest-4-cores
+    strategy:
+      matrix:
+        target: [build-runtime, build-runtime-fast]
+        package: [altair-runtime, centrifuge-runtime]
```

I was thinking about this and drew the conclusion that we don't need to build the WASM for `build-runtime-fast` but should add `development`runtime` as package for Demo
```suggestion
        target: [build-runtime]
        package: [altair-runtime, centrifuge-runtime, development-runtime]
```

---

DONE!